src/lwpMediaDevices.js

```diff
@@ -766,11 +839,49 @@ export default class extends lwpRenderer {
         previousTrack
       );
 
+      if (trackKind === "video" && preferedDevice.id === "screenCapture") {
+        return this.startScreenCapture();
+      }
+
+      if (this._captureStream) {
+        this.stopScreenCapture();
+      }
+
       if (previousTrack) {
         mutedInputs = previousTrack.enabled ? [] : [previousTrack.kind];
         this._removeTrack(mediaStream, previousTrack);
       }
 
+      if (trackKind === "video" && preferedDevice.id === "none") {
+        // Disable video for all streams, do not replace track or media stream
+        this._startedStreams.forEach((request) => {
+          if (request.mediaStream) {
+            request.mediaStream.getVideoTracks().forEach((track) => {
+              track.enabled = false;
+            });
+          }
+        });
+
+        this._availableDevices[preferedDevice.deviceKind].forEach(
+          (availableDevice) => {
+            if (availableDevice.id == "none") {
```

Comment:
is it intentional to use `==` instead of `===` ?

Reply:
My bad, it's a leftover form old code

Reply:
Fixed!